eng/pipelines/eng-common-sync.yml

```
@@ -30,16 +30,38 @@ stages:
         displayName: Sync ${{ parameters.DirectoryToSync }} Directory
 
         steps:
+        - ${{ if eq(variables['Build.Reason'], 'PullRequest') }}:
+          - pwsh: |
+              $patchDir = "$(Build.ArtifactStagingDirectory)/patchfiles"
+              New-Item $patchDir -Type Directory
+              $patchfiles = git format-patch HEAD...origin/master -- "${{ parameters.DirectoryToSync }}"
```

```suggestion
              $patchfiles = git format-patch HEAD...$(system.pullRequest.targetBranch)  -- "${{ parameters.DirectoryToSync }}"
```
This needs to be target branch in order to match the other logic in this file and to allow us to support other branches.